package.json

```diff
@@ -21,10 +22,12 @@
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.6",
     "react-native": "0.44.2",
+    "react-native-actionsheet": "^2.2.0",
     "react-native-communications": "^2.2.1",
     "react-native-device-info": "^0.10.2",
     "react-native-elements": "^0.9.3",
     "react-native-htmlview": "^0.9.0",
+    "react-native-material-design-searchbar": "^1.1.4",
```

Comment:
Is there something like an universal solution for search bar? I don't think it's a good idea to have split specific components for each platform that makes our code less mantainable. Also we don't want our search bar to depend on 2 different projects.
Hopefully that makes sense.